ci/infra/testrunner/README.md

```diff
@@ -463,7 +463,10 @@ optional arguments:
   -c, --cloud-provider  Use cloud provider integration
   -t TIMEOUT, --timeout TIMEOUT
                         timeout for waiting a node to become ready (seconds)
-
+  -m L L, --registry-mirror L L
+                        Add to the registry at the first L a mirror at the
+                        second L. Example:
```

Comment:
Maybe it's just me, but this reads a bit weird. From this help string I have a hard time to understand what the option is really doing.

Reply:
Agree it should maybe read something like `-m prefix mirror`

Reply:
How do I do that with the option library we use here? AFAIK it only supports one argument placeholder even if there are two.

Reply:
https://docs.python.org/3/library/argparse.html#metavar
```
parser = argparse.ArgumentParser(prog='PROG')
parser.add_argument('-x', nargs=2)
parser.add_argument('--foo', nargs=2, metavar=('bar', 'baz'))
parser.print_help()
usage: PROG [-h] [-x X X] [--foo bar baz]

optional arguments:
 -h, --help     show this help message and exit
 -x X X
 --foo bar baz
```

Reply:
Done. Is it now easy enough to understand?

Reply:
Yes, thanks!